Makefile

```diff
@@ -0,0 +1,74 @@
+SHELL := /usr/bin/env bash
+
+# build the registry image
+IMAGE_REGISTRY?=quay.io
+IMAGE_REPOSITORY?=openshift-sre
+IMAGE_NAME?=osd-operators
+CATALOG_NAMESPACE?=openshift-operator-lifecycle-manager
+DOCKERFILE?=./Dockerfile
+CHANNEL?=production
+GIT_SHA=$(shell git rev-parse HEAD | cut -c1-8)
+
+TEMPLATE_CS=templates/template_osd-operators.CatalogSource.yaml
+DEST_CS=manifests/00_osd-operators.CatalogSource.yaml
+
+SUBSCRIPTIONS=$(shell cat subscriptions.json)
+
+.PHONY: default
+default: build
+
+.PHONY: clean
+clean:
+	rm -rf manifests/
+
+.PHONY: manifests
+manifests:
+	mkdir -p manifests/
+	# create CatalogSource yaml
+	for TYPE in CatalogSource; do \
```

Comment:
Where does `CatalogSource` come from?

Reply:
We're defining the catalog source.  There's a template: `templates/template_osd-operators.CatalogSource.yaml`

Reply:
Oh, I see. It's treating it as a literal value! Will that list ever change, and if so, does it make sense to refer to it as a Makefile variable, like so?

```makefile

TEMPLATE_TYPES := CatalogSource

for TYPE in $(TEMPLATE_TYPES); do \
 ...
```

Reply:
Same here, I didn't get the benefit of having this static list with only one element, CatalogSource

IMO it's a little hard to read, cause you tend to look for a variable or list called CatalogSource